decidim-core/app/helpers/decidim/resource_helper.rb

```diff
@@ -75,7 +75,7 @@ def resource_locator(resource)
     # Returns a descriptive title for the resource
     def resource_title(resource)
       title = resource.try(:title) || resource.try(:name) || resource.try(:subject) || "#{resource.model_name.human} ##{resource.id}"
-      title = translated_attribute(title) if title.is_a?(Hash)
+      title = decidim_escape_translated(title) if title.is_a?(Hash)
```

Comment:
Seems like this change isn't in the original PR and neither in develop, so we should implement it there first just to be sure 

https://github.com/decidim/decidim/blob/a068bbb39f70d5b56c6ab0c4d4c0618c71d4270f/decidim-core/app/helpers/decidim/resource_helper.rb#L54

Reply:
We could implement it, but if it has not been added to the original PR, it may mean that this method is not being used anymore, as a result of redesign. On a quick search, i could not find any references of this method in develop `grep -RiHn 'resource_title(' ./`, yet, I could find 8 results on `release/0.27-stable`

Reply:
Nice catch. I've created the PR https://github.com/decidim/decidim/pull/12672 
Solving this conversation as we're handling it there